src/utils/utils.js

```diff
@@ -29,6 +29,18 @@ export const getComments = (setComments, setIsLoading, article_id, pageNumber) =
     }).catch(err => console.log(err))
 }
 
+export const changeVotes = (endpoint, votes, setVoteNum, setIsLoading) => {
+    setIsLoading(true)
+    const body = {
+        "inc_votes": votes
+    }
+    apiCall.patch(endpoint, body).then((response) => {
+        console.log(response.data.votes, 'votes')
+        setVoteNum(response.data.votes)
+        setIsLoading(false)
```

Comment:
This isn't making the most of optimistic rendering, since you're waiting for the patch to be made before resetting the visual. Take another look at the notes on Optimistic Rendering please!